app/org/sagebionetworks/bridge/models/surveys/WeightConstraints.java

```diff
@@ -0,0 +1,21 @@
+package org.sagebionetworks.bridge.models.surveys;
+
+import static org.sagebionetworks.bridge.models.surveys.UIHint.WEIGHT;
+
+import java.util.EnumSet;
+
+public class WeightConstraints extends DecimalConstraints {
+    private boolean isForInfant;
+
+    public WeightConstraints() {
+        super();
+    }
+
+    public void setIsForInfant(boolean isForInfant) {
+        this.isForInfant = isForInfant;
+    }
+
```

Comment:
It does, and currently is being inherited from DecimalConstraint. I may change this